subsys/bluetooth/host/hci_core.c

```diff
@@ -2423,27 +2423,27 @@ static void read_buffer_size_v2_complete(struct net_buf *buf)
 
 	BT_DBG("status %u", rp->status);
 
-	bt_dev.le.acl_mtu = sys_le16_to_cpu(rp->acl_mtu);
+	bt_dev.le.acl_mtu = sys_le16_to_cpu(rp->acl_max_len);
 	if (!bt_dev.le.acl_mtu) {
 		return;
 	}
 
-	BT_DBG("ACL LE buffers: pkts %u mtu %u", rp->acl_max_pkt,
+	BT_DBG("ACL LE buffers: pkts %u mtu %u", rp->acl_max_num,
 		bt_dev.le.acl_mtu);
 
-	max_num = MIN(rp->acl_max_pkt, CONFIG_BT_CONN_TX_MAX);
+	max_num = MIN(rp->acl_max_num, CONFIG_BT_CONN_TX_MAX);
```

Comment:
Unrelated to this change.

@Thalley Why is it min with CONN_TX_MAX here? It is not like that for v1, so looks quite suspicious.

Reply:
@joerchan I guess in case of a non-Zephyr controller, we might expect `CONFIG_BT_CONN_TX_MAX` in the host, but but will need to handle it if the controller returns a smaller value, or even vice versa. 

If controller responds with 5 and CONFIG_BT_CONN_TX_MAX=10, then the maximum number of buffers we can use is 5, and vice versa if the controller responds with 10 and CONFIG_BT_CONN_TX_MAX=5, then we have only allocated 5 buffers in the host I suppose.